nexus/src/app/mod.rs

```diff
@@ -79,7 +80,7 @@ pub struct Nexus {
     sec_client: Arc<steno::SecClient>,
 
     /// Task representing completion of recovered Sagas
-    recovery_task: std::sync::Mutex<Option<db::RecoveryTask>>,
+    recovery_task: Mutex<Option<db::RecoveryTask>>,
```

Comment:
Curious, why remove the `std::sync` here?  Just misc cleanup, or does this have some other impact?